node/src/cli/commands/login.command.ts

```diff
@@ -20,17 +20,19 @@ import { PlatformUtilsService } from "jslib-common/abstractions/platformUtils.se
 import { PolicyService } from "jslib-common/abstractions/policy.service";
```

Comment:
This class is still quite horrible, but I don't think that was the focus of your refactoring

Reply:
Please don't forget to validate DirectoryConnectors authentication when testing this

Reply:
LoginCommand is next on the hitlist, don't worry ;) especially as I have a really good understanding now of the auth flows.

But yes, I ruled it out of scope for now.